src/clj/game/cards/programs.clj

```diff
@@ -868,7 +867,8 @@
                   :waiting-prompt "Runner to make a decision"
                   :effect (req (let [from (take 5 (:deck runner))]
                                  (wait-for (reveal state side from)
-                                           (continue-ability state side (custsec-host from) card nil))))}
+                                           (continue-ability state side (custsec-host from) card nil)
+                                           (effect-completed state side eid))))}
```

Comment:
This is incorrect. Effect-completed should never be called after continue-ability; the ability being confused should call effect-completed.